app/controllers/admin/schools/participants_controller.rb

```diff
@@ -7,12 +7,7 @@ class Schools::ParticipantsController < Admin::BaseController
     before_action :set_school
 
     def index
-      @participants = User.where(id: @school.participant_profiles
-                                            .active
-                                            .ecf
-                                            .select(:user_id))
-                          .order(:full_name)
-                          .includes(participant_profile: %i[cohort school])
+      @participants = @school.participants.order(:full_name)
```

Comment:
Missing includes?

Reply:
This shows withdrawn participants, right?

Reply:
Missing? And what would you like to include in there? :P

a school? a school cohort? a cohort?

The technical decision of including NPQ profiles in ParticipantProfiles, coupled with those profiles looking nothing like ecf profiles means I can't include anything meaningful here...

Reply:
```
includes(
      early_career_teacher_profile: %i[cohort school],
      mentor_profile: %i[cohort school],
)
```